pupil-spa/src/app/check/check.component.ts

```diff
@@ -332,6 +334,7 @@ export class CheckComponent implements OnInit {
     const stateDesc = this.getStateDescription();
     console.log(`Refresh detected during state ${this.state} ${stateDesc}`);
     this.auditService.addEntry(new RefreshDetected());
+    this.appCountService.reset();
```

Comment:
If a physical refresh of the page happens, because `appUsageCounter` is not stored in the localStorage or anywhere else, it will get reset to 0, so this might be unnecessary - but it’s good to double-check.

Reply:
As it is stored in memory it will be automatically re-initialised to 0, as @dan points out.  Are there any cases where this would actually do something?